### PR TITLE
NRPT-395: update helper text and labels on collection page

### PR DESCRIPTION
**Pull request by mtCarto**
- [NRPT-395](https://bcmines.atlassian.net/browse/NRPT-395)
- [NRPT-398](https://bcmines.atlassian.net/browse/NRPT-398)
- [NRPT-397](https://bcmines.atlassian.net/browse/NRPT-397)

### CODE REVIEW

**Review by MaxWardle — APPROVED** [review]

File: angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.html [hunk 1]
```diff
@@ -31,6 +31,11 @@ <h2 class="border-0 mb-0">Edit Collection</h2>
           </div>
         </div>
         <div class="col-6 col-lg-4 col-xl-3">
+          <div class="d-flex justify-content-end text-left">
+            <span *ngIf="isEditing" class="grey-subtext d-block">
+              Cancel will cause you to lose any records or links added to this collection - if you do not want to lose the added items click update
```
Comment (KitArmstrong) [inline angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.html:36]: Not sure if we want a period on this or not.

**Review by KitArmstrong — APPROVED** [review]
👍

File: angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.html (hunk 1 above)
Comment (marklise) [reply angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.html:36]: ```Cancel will cause you to lose any records or links added to this collection.  If you do not want to lose the added items, click update.```

Comment (mtCarto) [reply angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.html:36]: 👍

**Pull request merged**